refinery/core/models.py

```diff
@@ -214,6 +214,11 @@ def duplicate_slug_check(self):
     def clean(self):
         # Check if model being saved/altered in Django Admin has a slug
         # duplicated elsewhere.
+
+        # Removes trailing/leading white spaces
```

Comment:
@drj11 My comment here was not clear. I will update it. The other reason for this method is to catch blank spaces passed as a slug (tested through the admin interface). This reason is why they are separated.